snappy/binaries.go

```diff
@@ -114,10 +113,10 @@ ubuntu-core-launcher {{.UdevAppName}} {{.AaProfile}} {{.Target}} "$@"
 		SnapArch:    arch.UbuntuArchitecture(),
 		SnapPath:    pkgPath,
 		Version:     app.Snap.Version,
-		UdevAppName: fmt.Sprintf("%s.%s", app.Snap.Name(), app.Name),
+		UdevAppName: fmt.Sprintf("snap.%s.%s", app.Snap.Name(), app.Name),
```

Comment:
there should be a helper in snap/ or on Info to make this,  having this kind of logic in many places that need to agree is what made such a pain to switch things around to start with